Cargo.toml

```diff
@@ -350,3 +350,7 @@ yansi = { version = "1.0", features = ["detect-tty", "detect-env"] }
 # alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "7fab7ee" }
 # alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "7fab7ee" }
 # alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "7fab7ee" }
+foundry-compilers = { git = "https://github.com/paritytech/foundry-revive-compiler.git", branch = "pkhry/simplify" }
```

Comment:
duplicated dependency?

Reply:
This is related to the version issue i mentioned in the group chat